index.html

```diff
@@ -5,9 +5,47 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Epilogue:wght@500;700&display=swap" rel="stylesheet">
+    <link rel="icon" type="image/x-icon" href="./images/favicon-32x32.png">
     <title>Intro section</title>
 </head>
 <body>
-    
+    <div class="wrapper">
+        <navbar class="navbarContainer">
```

Comment:
tu być może chodziło Ci o znacznik 'nav'

Reply:
Warto też rozważyć dodanie znacznika 'header' - nie jest to w sumie wymagane i niewiele zmieni ale warto wiedzieć, że taki znacznik występuje:
https://www.w3schools.com/tags/tag_header.asp

Reply:
i tak jak 'header' dotyczy całego nagłówka (w tym obrazków, tekstów itd) tak 'nav' dotyczy tylko nawigacji - elementów, które można kliknąć i gdzieś przejść dzięki temu